rbac/server/api/blocks.py

```diff
@@ -32,6 +33,37 @@
 
 
 @BLOCKS_BP.get("api/blocks")
+@doc.summary("API Endpoint to get all blocks")
+@doc.description("API Endpoint to get all blocks.")
+@doc.response(
```

Comment:
Should we use the `@doc.produces` decorator for 200 responses instead, or is `@doc.response` the preferred practice?

Reply:
Updated to use @doc.produces